config/profile/rhdh/plugin-infra/secret-setup.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
```

Comment:
let's rename secret-setup.sh to gitops-secret-setup.sh (or so) to make the intention more clear?